solar-system.rb

```diff
@@ -0,0 +1,89 @@
+
+class SolarSystem
+  attr_reader :ss_user_choice
```

Comment:
I don't think you need an `attr_reader` for `ss_user_choice`-- you've defined `ss_user_choice` as the name of a method. `attr_reader` helps make method definitions if you give it an instance variable.